docs/releases/5.1.txt

```diff
@@ -32,6 +32,19 @@ Minor features
 :mod:`django.contrib.admin`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+* The new ``LogEntry.objects.log_actions`` method allows for the bulk creation
+  of multiple LogEntry objects from a given queryset. This method was added as
+  a replacement for the deprecated log_action method, which was not optimized
+  for creating multiple LogEntry objects in bulk.
+
+* The new ``ModelAdmin.log_deletions`` method allows developers to log the
+  deletion of multiple objects at once by creating LogEntry objects for each
+  deleted object.
+
+* The ``ModelAdmin.log_change`` and the ``ModelAdmin.log_addition`` methods are
+  updated with the newly created ``LogEntry.objects.log_actions`` method
+  replacing the deprecated ``LogEntry.objects.log_action`` method.
```

Comment:
These are all undocumented private APIs. IMO release notes are not needed.

Reply:
Done.